Yona/Yona/nl.lproj/Localizable.strings

```diff
@@ -36,3 +36,11 @@
 "adddevice.user.infoText" = "Je hebt nog niet eerder ingelogd op dit apparaat. Ga naar je primaire device (waarschijnlijk je mobiel), open de Yona app, vervolgens naar instellingen en klik op ‘apparaat toevoegen’. Vul vervolgens onderstaande velden in.";
 
 "login.user.errorinfoText" = "Jouw account is geblokkeerd om veiligheidsredenen, gebruik de optie reset-toegangscode om jouw account te activeren.";
+
```

Comment:
Ok well make sure you update before the demo... I will make a note to get these checked too, we don't want bad dutch showing in the demo??